styles/stylesheet.css

```diff
@@ -0,0 +1,312 @@
+* {
+  box-sizing: border-box;
+}
+:root {
+  --green: #95be66;
+  --blue: #224971;
+  --grey: #e9e9e9;
+  --lightgrey: #e7e7e7;
+  --headerfont: 'Gelasio';
+  --textfont: 'Open Sans';
+}
+
+/* header */
+a {
+  text-decoration: none;
+  font-family: var(--textfont);
+}
+h1, h2, h3 {
+  color: var(--blue);
+  font-family: var(--headerfont);
+}
+header {
+  width: 100%;
+  background-image: url("../images/header-image.jpg");
+  background-size:cover;
+}
+.header-content-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  padding: 4rem 5rem 6rem 5rem;
+}
+.checkout-button-content-container > a {
+  /* border: dashed red; */
+  padding: .5rem 2rem;
+  background-color: var(--green);
+  color: white;
+  font-weight: bold;
+  font-size: .85rem;
+}
+.checkout-button-content-container,
+.cart-content-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  /* border: dashed red; */
+}
+.cart-content-container img {
+  padding-right: .5rem;
+  justify-content: flex-end;
+}
+.checkout-content-container {
+  /* margin: 5rem; */
+  grid-column-start: 3;
+  /* border: dashed purple; */
+}
+header + nav {
+  width: 100%;
+  background-color: var(--grey);
+}
+nav > ul {
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+}
+header + nav > ul {
+  /* padding: 1.5rem 10rem; */
+  padding: 1% 10%;
```

Comment:
be careful using percents with padding, it can sometimes look strange. Usually a set value would be fine. You could totally use rems for this and it'll look great.